machines/VerifiableCredential/VCItemMachine/VCItemMachine.ts

```diff
@@ -302,11 +302,11 @@ export const VCItemMachine = model.createMachine(
               onError: [
                 {
                   actions: [
-                    'setErrorAsWalletBindingError',
+                    'setErrorAsInvalidOtpError',
```

Comment:
We have to add condition to check the error message if it is a binding_auth_failed msg only then we will show invalid input error.